packages/nimble-components/build/generate-workers/source/matrix-renderer.ts

```diff
@@ -1,4 +1,5 @@
 import { expose } from 'comlink';
```

Comment:
Make sure prettier is run on this file, the prettierignore needs to be updated so that prettier runs on build/generate-workers/source

https://github.com/ni/nimble/blob/833a75fbdfcc23fb9a9f7964fef957818cb0e9d7/packages/nimble-components/.prettierignore#L12-L17

Reply:
I think the config change you made is a bit too aggressive; now we will be running prettier on any build output that's JSON or TS, which might include files generated by other tools that are never reviewed by humans. Can you exclude just the directory that is in source control, `build/generate-workers`?

Reply:
you are right, solved in this commit https://github.com/ni/nimble/pull/1929/commits/0486013906ea5f2169c549c9cf6e1d396840d2df